src/components/side-panel.js

```diff
@@ -214,6 +219,7 @@ export default function SidePanelFactory(
             {PanelComponent ? (
               <PanelComponent
                 datasets={datasets}
+                datasetUpdateNumbers={datasetUpdateNumbers}
```

Comment:
You don't need `datasetUpdateNumbers` if you use `copyTableAndUpdate` to update dataset color. you can import this util from `kepler-table.js`